doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -227,6 +227,7 @@ nRF5340 Audio
     The :ref:`net_buf_interface` will also contain the metadata about the audio stream in the ``user_data`` section of the API.
     This change was done to transition to standard Zephyr APIs, as well as to have a structured way to pass N-channel audio between modules.
   * The optional buildprog tool to use `nRF Util`_ instead of nrfjprog that has been deprecated.
+  * Buffer count (CONFIG_BT_ISO_TX_BUF_COUNT and CONFIG_BT_BUF_ACL_TX_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use-cases.
```

Comment:
```suggestion
  * The buffer count (CONFIG_BT_ISO_TX_BUF_COUNT and CONFIG_BT_BUF_ACL_TX_COUNT) to be in-line with SoftDevice Controller (SDC) defaults. This can be changed and optimized for specific use cases.
```

Reply:
In addition to Greg's suggestion, move the second sentence to the new line:
"This can be changed and optimized for specific use cases."

Reply:
It seems this suggestion is not implemented.